sound/soc/sof/intel/mtl.h

```diff
@@ -0,0 +1,77 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2020 Intel Corporation. All rights reserved.
```

Comment:
2020-2022 and i'd remove the author line

Reply:
ack